app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -64,7 +59,9 @@
     else controllers.incomeSources.cease.routes.DeclarePropertyCeasedController.show(ForeignProperty).url
 }
 
-@noneToUnknown(value:Option[String]) = @{value.getOrElse(getMessage("unknown"))}
+@noneToUnknown(value: Option[String]) = @{
```

Comment:
this would be applicable only to English, what about Welsh? might need to manipulate with Language resources here instead